packages/requisitions/src/RequestRequisition/api/api.ts

```diff
@@ -172,4 +175,21 @@ export const RequestRequisitionQueries = {
 
       throw new Error('Unable to create requisition');
     },
+  deleteRequisitions:
+    (api: RequestRequisitionApi, storeId: string) =>
+    async (requisitions: RequestRequisitionRowFragment[]) => {
+      const promises = requisitions.map(requisition => {
+        const input = requisitionParser.toDeleteInput(requisition);
+        return api.deleteRequestRequisition({ input, storeId });
+      });
+      const results = await Promise.all(promises);
+
+      const success = results.every(({ deleteRequestRequisition }) => {
+        return deleteRequestRequisition.__typename === 'DeleteResponse';
+      });
+
+      if (success) return results;
+
+      throw new Error('Could not delete requisition');
```

Comment:
maybe?
```suggestion
      throw new Error('Could not delete requisitions');
```
it's not clear how many couldn't be deleted 🤷